### PR TITLE
[Spaces] Changelog

### PR DESCRIPTION
**Pull request by cbensimon**

### CODE REVIEW

**Comment by osanseviero** [comment]
Looks good! Maybe we can have versions that would be mentioned in the top banner? (0.0.1, 0.0.2, etc)

**Review by osanseviero — APPROVED** [review]

File: docs/hub/spaces.md [hunk 1]
```diff
@@ -149,3 +149,11 @@ In the past few years, our team, in collaboration with other research groups, ha
...
 
 This project's goal is to experiment with an extensible way for users and organizations to host demos/apps on huggingface.co, in a more productized/scalable way than we’ve done in the past.
+
+## Changelog
+
+* [`2021-08-04`] - Add support for Debian packages
+* [`2021-08-03`] - Add support for Streamlit components
+* [`2021-08-03`] - Make Flax / Jax run smoothly on GPU
+* [`2021-08-02`] - Upgrade Streamlit to `0.82.0`
+* [`2021-08-01`] - Link to raw logs (build and container) from the space repository
```
Comment (julien-c) [inline docs/hub/spaces.md:159]:
```suggestion
* [`2021-08-04`] - Add support for Debian packages, add your `apt-get` dependencies into a `packages.txt` file
* [`2021-08-03`] - Add support for [Streamlit components](https://streamlit.io/components)
* [`2021-08-03`] - For GPU-activated Spaces, make sure Flax / Jax runs smoothly on GPU
* [`2021-08-02`] - Upgrade Streamlit to `0.82.0`
* [`2021-08-01`] - Link to raw logs (build and container) from the space repository (viewable by users with write access to a Space)
```

**Review by julien-c — APPROVED** [review]
suggestion to add a little more description to each change

File: docs/hub/spaces.md (hunk 1 above)
Comment (osanseviero) [reply docs/hub/spaces.md:159]: No strong feeling, but what do you think of going one step further:

```suggestion
* [`2021-08-04`]  v 0.0.4 - Debian packages support
  * You can now add your `apt-get` dependencies into a `packages.txt` file.
* [`2021-08-03`] v 0.0.3 - Streamlit components and Flax/Jax GPU improvements
  * Add support for [Streamlit components](https://streamlit.io/components)
  * For GPU-activated Spaces, make sure Flax / Jax runs smoothly on GPU
* [`2021-08-02`] v 0.0.2 - Upgrade Streamlit to `0.82.0`
* [`2021-08-01`] v 0.0.1 - Raw logs available.
  * Add link to raw logs (build and container) from the space repository (viewable by users with write access to a Space).
```

**Comment by cbensimon** [comment]
> Looks good! Maybe we can have versions that would be mentioned in the top banner? (0.0.1, 0.0.2, etc)

Not really sure we want to version HF Spaces. What do you think @julien-c ?

**Comment by julien-c** [comment]
Up to you!

Maybe the "version" is just the date (like you did in the changelog)

**Comment by cbensimon** [comment]
I don't think that we need versioning because users can only use the last / current version (unlike a pypi or npm pacakge).
Yes, I think the "version" is just the date in our case

File: docs/hub/spaces.md (hunk 1 above)
Comment (cbensimon) [reply docs/hub/spaces.md]: I like that proposal. Having a really short title plus bullet points for description

**Comment by cbensimon** [comment]
@julien-c @osanseviero what about the last proposal ? (Close to what @osanseviero proposed, but without version numbers)

**Comment by osanseviero** [comment]
Looks good to me! I would still suggest to group changes by date (e.g. there are to changes from 2021-08-03), but up to you. Thanks for taking care of this!

**Comment by cbensimon** [comment]
> Looks good to me! I would still suggest to group changes by date (e.g. there are to changes from 2021-08-03), but up to you. Thanks for taking care of this!

@osanseviero I prefer to keep one feature per line. I don't think that having multiple feature release per day will happen a lot

**Pull request merged**